### DISCUSSION TITLE
Create a 0.7.1 release?

### DISCUSSION THREAD

**Discussion by hunts** [NLnetLabs/domain#155] [General]
Hi @partim, 

Looks like we have a 0.8.0-dev under developed, I'm not sure about the schedule, but can I ask for a 0.7.1 release cut from https://github.com/NLnetLabs/domain/commit/065c15abede8204af960c0559615edd9e6aa9d5c? So that we can void using `[patch.crates-io]` for vendoring the forked version in our repos.

Thanks!

**Comment by partim** [comment]
That should be no problem at all. In fact, let me do that right away before I forget.

**Reply by hunts** [reply]
Thank you very much for the quick action!!

We got it,
```
cargo update -p domain@0.7.0                
    Updating crates.io index
    Updating domain v0.7.0 -> v0.7.1
```
